frontend/src/core/loaders/components/WaveLoader.css

```diff
@@ -0,0 +1,100 @@
+
+.overlay {
```

Comment:
Please make sure that everything in CSS files is encapsulated in the component's identified (for now an id, but with my proposed change, this would be in `.wave-loader`), so that it doesn't cause unexpected side effects in other components.